resources/assets/css/_tables.css

```
@@ -760,6 +760,110 @@
     @apply bg-transparent;
 }
 
+/* Compact tables */
+.table-container
+    table:not(.table-compact)
+    > tbody
+    > tr
+    > td.hoverable-cell
+    > div.table-cell-content,
+body:not(.table-compact)
+    .table-container
+    > tbody
+    > tr
+    > td.hoverable-cell
+    > div.table-cell-content {
+    @apply py-4;
+}
+
+.table-container
+    table.table-compact
+    > tbody
+    > tr
+    > td.hoverable-cell
+    > div.table-cell-content,
+body.table-compact
+    .table-container
+    table
+    > tbody
+    > tr
+    > td.hoverable-cell
+    > div.table-cell-content {
+    @apply py-3;
+}
+
+@screen sm {
+    .table-container
+        table.table-compact.table-compact-until-sm
+        > tbody
+        > tr
+        > td.hoverable-cell
+        > div.table-cell-content,
+    body.table-compact.table-compact-until-sm
+        .table-container
+        table
+        > tbody
+        > tr
+        > td.hoverable-cell
+        > div.table-cell-content {
+        @apply py-4;
+    }
+}
+
+@screen md {
+    .table-container
+        table.table-compact.table-compact-until-md
+        > tbody
+        > tr
+        > td.hoverable-cell
+        > div.table-cell-content,
+    body.table-compact.table-compact-until-md
+        .table-container
+        table
+        > tbody
+        > tr
+        > td.hoverable-cell
+        > div.table-cell-content {
+        @apply py-4;
+    }
+}
+
+@screen lg {
+    .table-container
+        table.table-compact.table-compact-until-lg
+        > tbody
+        > tr
+        > td.hoverable-cell
+        > div.table-cell-content,
+    body..table-compact.table-compact-until-lg
```

@alfonsobries I think I found the culprit 😅. If I remove the dot and recompile, it works!
```suggestion
    body.table-compact.table-compact-until-lg
```